packages/example-app-base/src/components/PropertiesTable/PropertiesTable.tsx

```diff
@@ -38,7 +79,8 @@ const DEFAULT_COLUMNS: IColumn[] = [
     maxWidth: 150,
     isCollapsable: false,
     isResizable: true,
-    isMultiline: true
+    isMultiline: true,
+    onRender: (item: IInterfaceProperty) => renderCell(item.type)
```

Comment:
Would be great to avoid these inline functions.

Reply:
I was trying to avoid declaring a separate function for each column, since each column needs to call `renderCell` with a different item property (type, name, description, etc.). It would be a small change to declare all the functions separately though.

Reply:
You could probably shorten this by using `.bind` to pass the property you need as an argument. However that would still create a new function. ¯\_(ツ)_/¯